inventory/byo/hosts.ose.example

```diff
@@ -410,6 +410,11 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Configure usage of openshift_clock role.
 #openshift_clock_enabled=true
 
+# Global OpenShift Environment Variables
+# Environment variables are added to /etc/sysconfig files for
+# each OpenShift service (node, master, api, controllers).
+#openshift_global_env_vars={"ENABLE_HTTP2": "true"}
```

Comment:
I don't really like this approach with just a single var for setting things across all services.

That said, I think there is room to have a var that sets things at a high level.

Thoughts on a top level var (OPENSHIFT_SERVICE_ENV_VARS), and then one for each service (maybe a var for applying to both master services). It would require merging the vars before applying, but it would allow for big and small hammers and allow for overriding specific vars on a specific service (or a subset of hosts with that service depending on where the vars are defined).